control/v1/control.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package control.v1;
+
+option go_package = "example/gen/control/v1;controlv1";
```

Comment:
Why is this needed?

Reply:
Will be removed by the merge to the `DaemonService` :-)